migrations/6_initialize_721token.js

```diff
@@ -3,16 +3,10 @@ const CodexTitle = artifacts.require('./CodexTitle.sol')
 const CodexTitleProxy = artifacts.require('./CodexTitleProxy.sol')
 
 module.exports = async (deployer, network, accounts) => {
+  const proxiedCodexTitle = CodexTitle.at(CodexTitleProxy.address)
```

Comment:
I'm surprised `CodexTitleProxy.address` works without first waiting for `CodexTitleProxy.deployed()`... how does it know the deployed address? From the previous migration steps I guess?

Reply:
Yeah, from the previous migration step. We were already doing this in one other location, just decided to do it here because it's a bit cleaner